client/src/components/events/normal_events.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
```

Comment:
file name should be the same as the default exported component! 
would EventCard.tsx be a better name..? I feel that Event.tsx might be a little too general and could cause confusion in the future :(